Tchap/Managers/Invite/InviteService.swift

```diff
@@ -120,7 +111,7 @@ final class InviteService: InviteServiceType {
     // Check whether a Tchap account has been created for this email. The closure returns a nil identifier when no account exists.
     private func discoverUser(with email: String, completion: @escaping (MXResponse<InviteServiceDiscoverUserResult>) -> Void) {
         let pid = MX3PID(medium: .email, address: email)
-        _ = self.identityService.lookup3PIDs([pid]) { response in
+        _ = self.session.identityService.lookup3PIDs([pid]) { response in
```

Comment:
Your current implementation doesn't satisfy "The closure returns a nil identifier when no account exists."
We will sync about in direct

Reply:
This is fixed in the last commit.